airbyte-cdk/python/airbyte_cdk/sources/streams/concurrent/cursor.py

```diff
@@ -18,19 +17,35 @@ def _extract_value(mapping: Mapping[str, Any], path: List[str]) -> Any:
     return functools.reduce(lambda a, b: a[b], path, mapping)
 
 
-class Comparable(Protocol):
+class GapType(Protocol):
```

Comment:
I don't understand this class name.

Reply:
This describes the gap between two cursor values. For example, datetime would have GapType = timedelta. I'll add a comment to explain that